src/components/CityColumnChart/CityColumnChart.scss

```diff
@@ -23,13 +19,21 @@
 .title {
   font-size: 22px;
   font-weight: 300;
-  margin-right: 7px;
+  margin: 0 7px 11px 0;
```

Comment:
This breaks the alignment of the (?) which got merged in yesterday:

![image](https://user-images.githubusercontent.com/4443482/32471397-2c1fe872-c3b1-11e7-9991-05666cdd7708.png)

You could wrap the title and the (?) in a div and give it the bottom margin. But I say let's get this merged and do that later.